geode-core/src/main/java/org/apache/geode/internal/SharedLibrary.java

```diff
@@ -129,107 +118,6 @@
     }
   }
 
-  /**
-   * @return true if this process is 64bit
-   * @throws RuntimeException if sun.arch.data.model doesn't fit expectations
-   */
-  public static boolean is64Bit() {
```

Comment:
Don't delete `is64Bit()` -- this method still has a purpose in AddressableMemoryManager without the native library.